exp/controllers/common_controller_test.go

```diff
@@ -1,109 +0,0 @@
-/*
```

Comment:
Are we ceding any important coverage by deleting this (and other) logging-related unit tests?

Reply:
I don't think so. The tests were not super effective, and as the original author, I'm no longer sold on the fact they verify anything of value.